src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1641,6 +1689,31 @@ namespace winrt::TerminalApp::implementation
     {
         _CloseAllTabs();
     }
+        
+    // Method Description:
+    // - Called when the primary button of the content dialog is clicked.
+    //   This calls _CloseFocusedTab(), which closes the tab currently
+    //   selected. This method will be called if
+    //   the user confirms to close the selected tab.
+    // Arguments:
+    // - sender: unused
+    // - ContentDialogButtonClickEventArgs: unused
+    void TerminalPage::_CloseWarningPrimaryButtonOnClick(WUX::Controls::ContentDialog /* sender */,
+                                                         WUX::Controls::ContentDialogButtonClickEventArgs /* eventArgs*/)
+    {
+        _CloseFocusedTab();
+    }
```

Comment:
This whole method looks like it got duplicated incorrectly

Reply:
Thank you. Resolved.